Proper indentation (#1)
blue-four-seven/endless-sky: data/human/hails.txt

		"uranium or uranium" "uranium"
		"neodymium and neodymium" "neodymium"
		"neodymium or neodymium" "neodymium"

phrase "friendly civilian"
	word
		"Be careful of"
		"A warning about"
		"Be suspicious of"
		"Don't trust"
	word
		" "
	word
		"those space casinos. I blew"
	word
		" "
	word
		"a"
		"two"
		"three"
		"four"
		"five"
	word
		" "
	word
		"million credits on "
	word
		"Bourne"
		"Spica"
		"Rust"
		"Starcross"
		"Skymoot"
		"Thunder"
		"Zug"
		"Flood"
		"Longjump"
		"Glaze"
		"Deep"
		"Clink"
		"Winter"
		"Solace"
		"Twinstar"
		"Sundrinker"
		"Trinket"
		"Dancer"
	word
		" "
	word
		"blackjack."
		"poker."
		"backgammon."
		"baccarat."
		"roulette."

phrase "friendly civilian"
	word
		"Have you read the latest edition of"
	word
		" "
	word
		"Cosmic Chronicles"
		"Stellar Spectacles"
		"Galactic Gazette"
		"Quantum Quandaries"
		"Nebula News"
		"Phantom Planets"
		"Vague Voyages"
		"Dark Dimensions"
		"Mysterious Moons"
		"Silent Satellites"
		"Zesty Zeniths"
		"Daring Discoveries"
		"Fiery Frontiers"
		"Global Galaxies"
		"Rapid Rockets"

> 		"Orbital Overload"

Author: ```suggestion
		"Orbital Overload"
		"Stars and Spaceships"
```

	word
		"? It says that"
	word
		" "
	word
		"pulse showers are good for your quantum pores."
		"exposing yourself to neodymium increases your risk of spacepox."

> 		"shoddily grown ion algae exhaust your psychic karma."

Author: ```suggestion
		"shoddily grown ion algae exhausts your psychic karma."
```

		"laser acupuncture lets you communicate with nebula ghosts."

> 		"star crystals bring luck with long distance relationships."

Author: ```suggestion
		"star crystals bring luck to long distance relationships."
```

> 		"consuming moon dust accelerates hair growth but causes lunar lunacy."

Author: ```suggestion
		"consuming moon dust accelerates hair growth but causes lunacy."
```

		"excessive exposure to zero gravity can lead to phantom limb syndrome."
		"alien prosthetics often come with unexpected telepathic abilities."
		"stardate tattoos increase your chances of alien abduction."

phrase "friendly civilian"
	word
		"I'm no weapons expert, but it feels better firing"
	word
		" "
	word
		"energy blasters"
		"particle cannons"
		"proton guns"
	word
		" "
	word
		"than"
		"compared to"
	word
		" "
	word
		"beam lasers."

> 		"electron beams."

Author: ```suggestion
```

		"heavy lasers."

phrase "friendly civilian"
	word
		"I shouldn't have sold my old ship. It was a"
	word
		" "
	word
		"Shuttle"
		"Star Barge"
		"Sparrow"
		"Heavy Shuttle"
		"Freighter"
		"Fury"
	word
		" "
	word
		"that never had"
	word
		" "
	word
		"stalling engines."

> 		"broken batteries."

Author: ```suggestion
```

		"unstable shielding."
		"overheating fuel cells."
		"a glitchy HUD."
		"faulty ship detection."

phrase "friendly civilian"
	word
		"Finished cleaning my cargo hold. The" 
		"Cargo hold is all spick and span. The"
		"Finished scrubbing the hold. The"
	word
		" "
	word
		"freshener" 
		"perfume"
		"spray"
		"fragrance"
	word
		" "
	word
		"I used makes it smell like"
	word
		" "
	word
		"strawberries!"
		"bananas!"
		"peaches!"
		"roses!"
		"jasmine!"
		"lavender!"

phrase "friendly civilian"
	word
		"Have you seen the movie where the" 
	word
		" "
	word
		"bad guy"
		"villain"
	word
		" "
	word

> 		"mounted modded twin blasters on a modded Argosy"

Author: ```suggestion
		"mounts overcharged twin blasters on a modded Argosy"
```
